src/app/pages/checkout-payment/payment-parameter-form/payment-parameter-form.component.ts

```diff
@@ -28,7 +28,17 @@ export class PaymentParameterFormComponent implements OnInit, OnChanges {
   ngOnInit() {
     this.fields = [];
     this.paymentMethod.parameters.forEach(param => this.fields.push({ ...param }));
-    this.fields.forEach(field => (this.model[field.key as string] = field.props?.options ? undefined : ''));
+    this.fields.forEach(field => {
+      if (field.key === 'IBAN') {
+        field.props.mask = 'UU00 AAAA 0000 0009 9999 9999 9999 9999 99';
```

Comment:
There is no ngx-mask mask for an optional number or letter. That's why we decided to use the mask for an optional number, as this matches for IBAN pattern in most countries.
The generation of white space by inserting letter in case of an optional number is a bug in ngx-mask.